pages/consortium.html

```diff
@@ -0,0 +1,140 @@
+<#def TITLE>The Coq Consortium</#def>
+<#include "incl/header.html">
+
+<div class="framework">
+<div class="frameworklabel">A pool of resources and a platform for exchanges</div>
+<div class="frameworkcontent">
+
+<p>The objective of the Consortium is to make the community of users feel empowered with respect to
+    the future of the Coq system, giving them opportunities to see how the system will evolve, where it is
+    going, and to take an active part in this process. On the one hand, we wish to establish <strong>transparent
+    ways of contributing financially and technically</strong> to the development of the system; on the other
+    hand we wish to make sure that the <strong>most pressing needs of the user community are addressed
+    efficiently</strong> by providing a fantastic working environment to some of the most competent developers
+    trained in our teams.</p>
+
+</div>
+
+</div>
+
+<div class="framework">
+<div class="frameworklabel">Support services</div>
+<div class="frameworkcontent">
+    <p>The Consortium provides a package of support services:
+        <ul>
+            <li>Premium support for bugs. Reports by Consortium members will be processed in priority by
+the Consortium engineers. Private bug reports can be processed by Consortium engineers
+                under an NDA to explore a code base and minimize a bug example.</li>
+            <li>A session of on-site training each year. (Optional, travel and lodging expenses should be paid
+                by the member).</li>
```

Comment:
I believe "accommodation" will be more idiomatic than "lodging"

Reply:
I'd rather have confirmation from native speakers, but it sounds to me as a difference between American and British English.

Reply:
This was written in Andrew's letter, so I'm pretty sure it is idiomatic English :)

Reply:
okay then